src/Components/CreateImageWizard/CreateImageWizard.scss

```diff
@@ -10,7 +10,7 @@
 
 .pf-c-dual-list-selector {
     --pf-c-dual-list-selector__menu--MinHeight: 18rem;
-    --pf-c-dual-list-selector__menu--MaxHeight: 18rem;
+    --pf-c-dual-list-selector--GridTemplateColumns--pane--MinMax--max: 100vw;
```

Comment:
Do I understand right, that this should help to "stretch" the dual list selector to fix the Wizard height wise? Seem to be stuck on the MinHeight for me (Mozilla)

Reply:
This is what makes the dual list selector fill up the entire _width_ of the page.

I removed the MaxHeight because it wasn't doing anything useful.

The MinHeight is the same as before... it is a bit larger than the default. Theoretically we should be able to set it to `100vh` just like the width is `100vw` but it isn't so easy (you can try yourself if you want)... possibly related to Data Driven Forms, it ends up getting the wrong size and is way too tall. So for now, I'm just keeping it at 18 rem... the size it's always been.